src/components/new-time-entry/NewTimeEntry.tsx

```diff
@@ -1,10 +1,10 @@
 import React, { useContext, useRef, useState } from "react";
+import { useMutation } from "@apollo/client";
+import { ADD_TIME_ENTRY } from "../../GraphQL/Mutations";
```

Comment:
I would put this line with the Styles and the Types as the upper two imports are external